package/MDAnalysis/analysis/encore/clustering/ClusteringMethod.py

```diff
@@ -195,6 +195,9 @@ def __init__(self,
                 (default is 50). Parameter in the Affinity Propagation for
                 clustering.
 
+            **kwargs : Other keyword arguments are passed to : class:
```

Comment:
```suggestion
            **kwargs (optional): Other keyword arguments are passed to :class:
```

You can't have a space in the `:class:` tag for Sphinx.


Anyway, oops, that didn't turn out looking so good. https://mdanalysis--3178.org.readthedocs.build/en/3178/documentation_pages/analysis/encore/clustering.html?highlight=affinitypropagation#MDAnalysis.analysis.encore.clustering.ClusteringMethod.AffinityPropagation

Reply:
The above changes were not producing the desired result. I have made the changes according to me. Please have a look at it! In my opinion, it looks good. 

Docs build: https://mdanalysis--3178.org.readthedocs.build/en/3178/documentation_pages/analysis/encore/clustering.html

Reply:
Thanks, that looks much better.